### ISSUE TITLE
Weird bug while using the storage system

### ISSUE THREAD

**Issue by takeshipoggies** [blushiemagic/MagicStorage#229]
I'm very sorry if this type of issue was already posted, I didn't find anything like it.
I'm experiencing this weird bug while searching for items in my storage system.
I input the '#' key to search for item tooltips in the storage, and then as soon as I input the character 'f', the game not exactly freezes, but
- The music stops playing,
- The camera slightly shifts off-center, then immediately re-centers,
- The cursor stops moving, and in general the mouse stops responding in-game,
- All the keys stop working (I can't move nor interact with anything),
- A bunch of things stop moving (the clouds, the background, NPCs, critters,...),
- Texture bugs happen, for some items like placed Calamity Infernum boss relics, and it looks like the sprites for fire particles that appear throughout the animation are present all together and frozen in place (probably worth noting, I don't know),
- Apparently the time in-game stops flowing.
I say the game not exactly freezes because some things - I mostly noticed torches, the particles and changes in environmental illumination from torches - still work and move normally, the game just enters this semi-frozen state and I can't seem to solve it manually if not by quitting the game.

I don't know what else to do because the game effectively didn't crash, so there's no crash log.
It could be some debug mode freeze hotkey that I didn't know, but alas, this only works if I type inside the Storage Heart text box.

Thank you in advance for helping me :D

**Comment by DeiusShrab** [comment]
I've also run into this issue, with the character 'u' after typing '#' into the storage heart. 'u' is not bound to anything in my Keybindings.

Installed Mods:
- MaxStackExtra
- ShorterRespawn
- MagicStorage
- CheatSheet
- DialogueTweak
- WMITF
- StarsAbove
- AutoTrash
- OreExcavator
- WingSlotExtra
- AlchemistNPCLite
- RecipeBrowser
- BossChecklist
- Census
- SubworldLibrary
- OreSeeds
- BossCursor
- CalamityInterludes
- CalamityMod
- CalamityModMusic
- CalValEX
- CalamityFlamethrowers
- InfernumMode
- miningcracks_take_on_luiafk
- UnofficialCalamityWhips
- AllTheWalls
- ProvidenceNightTheme
- SmarterCursor
- BetterBossBar

**Comment by ExterminatorX99** [comment]
Does it eventually unfreeze if you leave it long enough?

**Comment by DeiusShrab** [comment]
No, I ended up having to force-quit the game every time

**Comment by ExterminatorX99** [comment]
Does this issue also happen when there's a small number of items in the storage?

**Comment by DeiusShrab** [comment]
Nope, the issue didn't appear when I put a handful of items in
I did manage to snag a client.log of the issue, it may have something to do with the tooltip on an Infernum item? The log file gets spammed with an OutOfRange Exception
[client.log](https://github.com/blushiemagic/MagicStorage/files/11267467/client.log)

**Comment by ExterminatorX99** [comment]
That is likely the issue. Can you try if it stills freezes with Infernum disabled?
Don't worry about any of the items, tml will keep them safe even with the mod unloaded.

**Comment by ExterminatorX99** [comment]
@takeshipoggies Do you also use the Infernum mod?

**Comment by DeiusShrab** [comment]
Yup. Disabling the Infernum mod does fix the issue, and it returns when Infernum is re-enabled.

**Comment by ExterminatorX99** [comment]
That means it's an issue with the Infernum mod. Please report it to them

**Comment by absoluteAquarian** [comment]
The following message has been copy/pasted to all issues older than 3 months.

In order to clean up how the project looks, I'll be closing all old issues.  If you believe that this issue is still relevant, please make a new issue that references this issue in its message.
